xml/System.Windows.Forms/WebBrowserBase.xml

```diff
@@ -2615,7 +2615,15 @@
       <Docs>
         <param name="m">The windows <see cref="T:System.Windows.Forms.Message" /> to process.</param>
         <summary>This member overrides <see cref="M:System.Windows.Forms.Control.WndProc(System.Windows.Forms.Message@)" />.</summary>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+Certain messages are forwarder directly to the ActiveX control, others are first processed by the wndproc of Control
```

Comment:
Is this information that the average MS Docs user would be interested in, or is this aimed to the maintainers of the source code?

```suggestion
Certain messages are forwarder directly to the ActiveX control, others are first processed by the wndproc of Control.
```